Tests/LibWeb/Text/input/IntersectionObserver/implicit-root-after-originating-iframe-is-removed-from-dom.html

```diff
@@ -0,0 +1,13 @@
+<body>
+<iframe></iframe>
+<script src="../include.js"></script>
+<script>
+    let iframe = document.querySelector('iframe');
+    iframe.src = "data:text/html," + encodeURI("<script>new IntersectionObserver(function() { }, { })<" + "/script>");
+    iframe.onload = function() {
+        iframe.remove()
+        if (internals)
+            internals.signalTextTestIsDone();
+        println("PASS if we didn't crash!");
+    };
```

Comment:
Is it worth wrapping this in an `asyncTest` to avoid exposing `internals`?

```js
asyncTest(done => {
    let iframe = document.querySelector('iframe');
    iframe.src = "data:text/html," + encodeURI("<script>new IntersectionObserver(function() { }, { })<" + "/script>");
    iframe.onload = function() {
        iframe.remove()
        done();
        println("PASS if we didn't crash!");
    };
});
```

Reply:
Yeah, let's do that :)